bedrock/firefox/templates/firefox/ios.html

```diff
@@ -122,6 +124,38 @@
   </div>{#-- /.container --#}
 </section>{#-- /#sync --#}
 
+<section class="section section-privacy" id="privacy">
+  <div class="container">
+    <h2 class="heading">{{ _('Keep your private info private') }}</h2>
+
+    <p class="lead">
+      {{ _('We believe your personal information belongs to you.') }}
+      {{ _('Firefox for iOS is filled with features to help keep it that way. ') }}
+    </p>
+
+    <ul>
+      <li id="private-browsing">
+      {% if waffle.switch('tracking-protection') %}
+        <h3>{{ _('Private Browsing with<br> Tracking Protection') }}</h3>
+      {% else %}
+        <h3>{{ _('Private Browsing') }}</h3>
+      {% endif %}
+        <p>{{ _('Browse without having your history, searches or cookies saved, and block hidden third parties from following you across the Web.') }}</p>
```

Comment:
This string mentions blocking third parties, which needs clarification too I guess.